projects/1.20/assets/neapolitan/neapolitan/lang/zh_cn.json

```
@@ -0,0 +1,310 @@
+{
+  "block.neapolitan.vanilla_ice_cream_block": "香草冰淇淋块",
+  "block.neapolitan.chocolate_ice_cream_block": "巧克力冰淇淋块",
+  "block.neapolitan.strawberry_ice_cream_block": "草莓冰淇淋块",
+  "block.neapolitan.banana_ice_cream_block": "香蕉冰淇淋块",
+  "block.neapolitan.mint_ice_cream_block": "薄荷冰淇淋块",
+  "block.neapolitan.adzuki_ice_cream_block": "红豆冰淇淋块",
+
+  "block.neapolitan.milk_cauldron": "装有奶的炼药锅",
+  "block.neapolitan.vanilla_milkshake_cauldron": "装有香草奶昔的炼药锅",
+  "block.neapolitan.chocolate_milkshake_cauldron": "装有巧克力的炼药锅",
+  "block.neapolitan.strawberry_milkshake_cauldron": "装有草莓奶昔的炼药锅",
+  "block.neapolitan.banana_milkshake_cauldron": "装有香蕉奶昔的炼药锅",
+  "block.neapolitan.mint_milkshake_cauldron": "装有薄荷奶昔的炼药锅",
+  "block.neapolitan.adzuki_milkshake_cauldron": "装有红豆奶昔的炼药锅",
+
+  "block.neapolitan.chocolate_block": "巧克力块",
+  "block.neapolitan.chocolate_bricks": "巧克力砖",
+  "block.neapolitan.chocolate_brick_slab": "巧克力砖台阶",
+  "block.neapolitan.chocolate_brick_stairs": "巧克力砖楼梯",
+  "block.neapolitan.chocolate_brick_wall": "巧克力砖墙",
+
+  "block.neapolitan.chiseled_chocolate_bricks": "雕纹巧克力砖块",
+
+  "block.neapolitan.chocolate_tiles": "巧克力瓦",
+  "block.neapolitan.chocolate_tile_slab": "巧克力瓦台阶",
+  "block.neapolitan.chocolate_tile_stairs": "巧克力瓦楼梯",
+  "block.neapolitan.chocolate_tile_wall": "巧克力瓦墙",
+
+  "block.neapolitan.frond_thatch": "蕉叶茅块",
+  "block.neapolitan.frond_thatch_stairs": "蕉叶茅楼梯",
+  "block.neapolitan.frond_thatch_slab": "蕉叶茅台阶",
```

```suggestion
  "block.neapolitan.frond_thatch": "蕉叶茅草块",
  "block.neapolitan.frond_thatch_stairs": "蕉叶茅草楼梯",
  "block.neapolitan.frond_thatch_slab": "蕉叶茅草台阶",
```
算是和abnormals系列的其他thatch统一翻译